app/src/main/res/values-v21/styles.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="PumpkinTheme" parent="Theme.AppCompat.Light.DarkActionBar">
```

Comment:
I decided to keep v21 styles as some functions, like status bar transparency (needed for sidenav) only presents in Lollipop.